### DISCUSSION TITLE
Windows Server 2019 supported?

### DISCUSSION THREAD

**Discussion by R3alM0m1X82** [HotCakeX/Harden-Windows-Security#446] [Q&A] [answered]
Congratulations for a great program, I can't get it to work on Windows Server 2019 (1809) is it supported? I ran the powershell command inline but got the error AppControl: Could not find a parameter matching the name 'DeferRegistrationWhenPackagesAreInUse'

**Comment by HotCakeX** [comment, marked as answer]
Hi, thank you!

Here are the supported OSes:

* Windows 11 24H2
* Windows 11 23H2
* Windows Server 2025

Just added them to the documentation as well
https://github.com/HotCakeX/Harden-Windows-Security/wiki/AppControl-Manager#supported-operation-systems

<br>

I'm trying to make AppControl Manager supported on Home edition of those OSes too, stay tuned for it!

<br>

**Reply by R3alM0m1X82** [reply]
Thanks a lot for quick reply! ;)
